src/loops/simple_programs_with_repitition/hard/q5/NowThatsPerfect.java

```diff
@@ -0,0 +1,36 @@
+package loops.simple_programs_with_repitition.hard.q5;
+
+/*
+Write a program with the class name "NowThatsPerfect" to accept a number from the user and determine whether it is a "Perfect" number.
+A perfect number is one whose factors sum up to equal the given number.
```

Comment:
As this is a java question, we also need to also provide what they need to print if the number is a perfect number. I would just edit the question to say something like "If the number is a perfect number, print x is a Perfect Number!".